src/editor.tsx

```diff
@@ -109,13 +132,59 @@ function EditInner({
 
   return (
     <>
+      {renderSaveVersionModal()}
       {renderEditToolbar()}
       <Layout>{children}</Layout>
       {renderExtraEditorStyles()}
       <div ref={formDiv} />
     </>
   )
 
+  function renderSaveVersionModal() {
+    return (
+      <Modal
+        isOpen={saveVersionModalIsOpen}
+        onRequestClose={() => setSaveVersionModalIsOpen(false)}
+        style={{
+          content: {
+            top: '50%',
+            left: '50%',
+            bottom: 'auto',
+            right: 'auto',
+            transform: 'translate(-50%, -50%)',
+          },
+        }}
+      >
+        <p>Erstelle und speichere eine neue Version:</p>
+        <input
+          type="text"
+          className="mt-3 border-gray-300 focus-visible:outline-sky-800 rounded-md p-2 block w-full border"
+          ref={commentInput}
+          size={50}
+          placeholder="Name der neuen Version"
+        />
+        <div className="text-right mt-3">
+          <button
+            className="inline-block rounded-md p-2 mr-2 text-white bg-sky-800"
+            onClick={async () => {
+              await save(commentInput.current?.value)
+              setIsFirstSave(true)
+              setSaveVersionModalIsOpen(false)
+            }}
+          >
+            Speichern
+          </button>
+          <button
+            className="inline-block rounded-md p-2 border text-sky-800 border-gray-500"
+            onClick={() => setSaveVersionModalIsOpen(false)}
+          >
+            Schließen
+          </button>
+        </div>
+      </Modal>
```

Comment:
@elbotho I need your help here in the styling. Currently it looks like: 
![2022-09-30-180915_558x274_scrot](https://user-images.githubusercontent.com/1327215/193312080-2414d48f-a306-4c92-b70d-817c14ab4f51.png)